modules/ROOT/partials/events/advtemplate-events.adoc

```diff
@@ -0,0 +1,9 @@
+The following events are provided by the xref:advanced-templates.adoc[Advanced Templates plugin].
+
+[cols="1,1,2",options="header"]
+|===
+|Name |Data |Description
+|NewTemplate |N/A |Fired when a new template is added.
+|===
+
+NOTE: This is a pure guess. I don’t know if this (or any) event exists for this plugin; if the event name is correct if it does exist; or if it works as described.
```

Comment:
I can't find any events in the advtemplate repo. Slightly concerning but you should ask Husky team about that.

Reply:
Right, no events added

Reply:
Are there any plans to add events? Particularly for when a template is added or inserted.

Reply:
1. Removed `advtemplate-events.adoc` from `/partials/events/`.

2. Removed include statement pointing to `advtemplate-events.adoc` from `advanced-templates.adoc`.